app/models/null_court.rb

```diff
@@ -0,0 +1,26 @@
+class NullCourt
+  include ActiveModel::Model
```

Comment:
Is this needed anymore? Wasn't this just added to try to fix the issue linking to a NullCourt.